english/git_disadvantage

```diff
@@ -0,0 +1 @@
+Subproject commit 6c02811a9f7c981ee335145a6e4dbb3242e24a9c
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.